### PR TITLE
Add tests for comment and home views

### LINKED ISSUE #639: Test other views
Currently testing only /examine in #632

### PR DESCRIPTION
**Pull request by ihodes**
fixes #639 (Except for tasks—should we leave the issue open for that? Also doesn't test about… but that is just rendering a static page for now)

[Review on Reviewable](https://reviewable.io/reviews/hammerlab/cycledash/643)

### CODE REVIEW

File: tests/python/test_views.py [hunk 1]
```diff
@@ -54,3 +60,49 @@ def test_examine(self, *mocks):
...
+        asserts.eq_(len(bams), 2)
+
+        asserts.eq_(len(last_comments), 2)
+        comment2_found = find(last_comments, lambda c: c['vcfId'] == run['id'])
+        asserts.eq_(comment2_found['commentText'], comment2['comment_text'])
+
+
+    @mock.patch('cycledash.views.render_template', mocked)
+    def test_comments(self, *mocks):
+        comment1 = create_comment_with_text(self.run['id'], 'this is some text')
```
Comment (tavinathanson) [inline tests/python/test_views.py]: Neat, there's a whole comments testing API in Python now? I owe you!

File: tests/python/test_views.py [hunk 2]
```diff
@@ -35,11 +38,14 @@ def setUp(self):
             self.run = dict(res.fetchone())
 
     def tearDown(self):
-        with tables(db.engine, 'projects', 'bams', 'vcfs') as (con, projects, bams, runs):
+        with tables(db.engine, 'projects', 'bams', 'vcfs', 'user_comments') \
```
Comment (tavinathanson) [inline tests/python/test_views.py]: Weird, I thought I posted this comment on another PR but maybe not: anyway, maybe turn this `delete_tables(comments, runs, bams, projects)` into a helper?

**Comment by tavinathanson** [comment]
Sorry for the delay! Looks good. Merge at will. Maybe close that issue and open up a new one called "Test tasks"?

File: tests/python/test_views.py (hunk 2 above)
Comment (ihodes) [inline tests/python/test_views.py]: Ah yes you did on the other test PR; I'll do it on this one, and then take care of porting it to the other tests after these PRs are merged in?

File: tests/python/test_views.py (hunk 1 above)
Comment (ihodes) [inline tests/python/test_views.py]: The API is almost totally covered! Yay!

**Comment by ihodes** [comment]
Good idea, thanks for the review!

File: tests/python/test_views.py (hunk 2 above)
Comment (tavinathanson) [inline tests/python/test_views.py]: :+1:

**Pull request merged**